<launch>
  <include file="$(find zed_cpu_ros)/launch/zed_cpu_ros.launch" ns="camera"/>
  <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" respawn="true" output="screen" ns="camera">
<!--   	<remap from="image_raw" to="/left/image_raw"/>
  	<remap from="camera_info" to="/left/camera_info"/> -->
  </node>

</launch>